energy-integration/farm-boosted-yields/src/lib.rs

```diff
@@ -3,12 +3,15 @@
 elrond_wasm::imports!();
 elrond_wasm::derive_imports!();
 
+use core::cmp;
+
 use common_types::{Nonce, PaymentsVec};
 use week_timekeeping::Week;
 use weekly_rewards_splitting::{base_impl::WeeklyRewardsSplittingTraitsModule, ClaimProgress};
 
 const MAX_PERCENT: u64 = 10_000;
 const BLOCKS_PER_WEEK: u64 = 100_800u64;
+pub const COLLECT_UNDISTRIBUTED_BOOSTED_REWARDS_WEEK_OFFSET: usize = 4;
```

Comment:
let's be safe and change to 5.

Reply:
The algorithm uses this constant in an exclusive manner, meaning that the rewards cannot be collected if the number of passed weeks is not exclusively greater than 4 (as shown in the test). Changing it to 5 would result in a claim logic inconsistency.